### ISSUE TITLE
AirPlay 2

### ISSUE THREAD

**Issue by donavanbecker** [BenSova/Mini-Monterey-Patcher#2]
Any possibility to enable Airplay 2 to Mac? Or to add this to your patcher once you release it?

**Comment by Monkiey** [comment]
The Dev will try to enable it in Patched Monterey

**Comment by BenSova** [comment]
I tried, it didn't work out, but I will have it in Patched Monterey.

**Comment by donavanbecker** [comment]
> I tried, it didn't work out, but I will have it in Patched Monterey.

https://khronokernel.github.io/macos/2021/06/22/AIRPLAY.html

**Comment by Monkiey** [comment]
Hello, the link where you got that info is for OCLP an entirely different macos patcher which does have this feature because it uses open core which is a different patching method macOS that makes it possible to enable airplay if you want to use it to get airplay here is more info: https://github.com/dortania/OpenCore-Legacy-Patcher

**Comment by donavanbecker** [comment]
@Monkiey, I would rather wait. wasn't sure if you were aware or not so figured i would share.

**Comment by Monkiey** [comment]
Yes I am aware of it and I actually installed oclp and tried and it’s amazing like apple totally should enable it for older macs

**Comment by Monkiey** [comment]
By they way still prefer patched sur and it’s simplicity

**Comment by donavanbecker** [comment]
@Monkiey, me also
